### DISCUSSION TITLE
mapbox-gl-draw plugin example not working.

### DISCUSSION THREAD

**Discussion by 21F0rc3** [maplibre/maplibre-gl-js#2594] [Q&A]
Hi, i have noticed that the example in the docs of how to draw polygon on maplibregl(https://maplibre.org/maplibre-gl-js-docs/example/mapbox-gl-draw), using mapbox-gl-draw plugin is not working. Can someone give me an update, is this somthing you already know about and if it is going to be fixed?

Thanks as always :)

**Comment by 21F0rc3** [comment]
Update: On a personal project i changed maplibre-gl to version 2.4.0, and mapbox-gl-draw to 1.2.0 and works as before. It seems that was caused by either an update of the mapbox-gl-draw or the maplibre-gl.

**Reply by HarelM** [reply]
We are not aware of it, but we do not maintain the draw plugin either, I would recommend opening an issue there and see if there's anything that needs to be done in order to fix this, a PR to the docs repo would be the ideal solution if the problem is with the docs...
Thanks for taking the time to report this!
